### ISSUE TITLE
BUG: Indentation in code blocks is sometimes missing

### ISSUE THREAD

**Issue by danieltomasz** [obsidianmd/obsidian-clipper#145]
**Version (please complete the following information):**
 - OS: [e.g. Windows] MacOS
 - Browser [e.g. Chrome, Firefox] Arc 
 - Web Clipper version: [e.g. 0.5.0] )0.9.4 (latest from chrome plugin shop)
- Obsidian version: [e.g. 1.7.2] 1.7.5

Code in medium post has wrong formatting and intendation isn't saved; (in locked and unlocked post)
For example here https://medium.com/@LearnPythonProgramming/exploratory-data-analysis-eda-with-python-a-hands-on-guide-ef5e20484ba8
following code 
```
import pandas as pd

# Load the dataset
data = pd.read_csv('movies.csv')

# Display the first few rows
print(data.head())
```
 is  added as 
```
import pandas as pddata = pd.read_csv('movies.csv')print(data.head())
``` 
into markdown file

**Comment by Alezy80** [comment]
The same issue on Habr.com. Look code at the middle of the [post](https://habr.com/en/articles/856126/)

**Comment by aquinary** [comment]
Same problem

**Comment by crosfield** [comment]
Confirm the problem on habr.com

**Comment by doraemonlei** [comment]
Same problem on WeChat public account article

**Comment by kepano** [comment]
Should be fixed in [0.10.6](https://github.com/obsidianmd/obsidian-clipper/releases/tag/0.10.6)

**Comment by dreznicek** [comment]
This is waaaay better! Thanks you, my coding friends!
